cpp/arcticdb/util/test/test_storage_lock.cpp

```diff
@@ -242,3 +285,39 @@ TEST(StorageLock, ForceReleaseLock) {
     // Clean up locks to avoid "mutex destroyed while active" errors on Windows debug build
     first_lock.unlock(lock_data->store_);
 }
+
+TEST(StorageLock, OptimisticForceReleaseLock) {
+    // This test does the same as ForceReleaseLock, but testing the try_lock() method instead of lock()
+    // Since lock() implements the retry logic but try_lock doesn't, the OptimisticForceReleaseLockTask
+    // will call try_lock() repeatedly until the TTL expires and it can obtain the lock. A timeout is implemented
+    // in the task so this doesn't hang forever in case of an error.
+    SKIP_MAC("StorageLock is not supported");
+    using namespace arcticdb;
+
+    auto lock_data = std::make_shared<LockData>(4);
+    folly::FutureExecutor<folly::CPUThreadPoolExecutor> exec{4};
+    // This is set in milliseconds => 50ms for the preempting check
+    ConfigsMap::instance()->set_int("StorageLock.WaitMs", 50);
+    // This is set in nanoseconds => 200ms for the TTL
+    ConfigsMap::instance()->set_int("StorageLock.TTL", 200 * 1000 * 1000);
+
+    std::vector<std::shared_ptr<StorageLock<>>> locks;
```

Comment:
Where is this used?

Reply:
Ah thanks. I noticed it was unused and deleted only one of the two places.